tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -195,6 +195,41 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		return restConfig, storageclass, profileID
 	}
 
+	staticProvisioningPreSetUpUtilForVMDKTests := func(ctx context.Context) (*restclient.Config, *storagev1.StorageClass, string) {
+		log := logger.GetLogger(ctx)
+
+		namespace = getNamespaceToRunTests(f)
+		// Get a config to talk to the apiserver
+		k8senv := GetAndExpectStringEnvVar("KUBECONFIG")
+		restConfig, err := clientcmd.BuildConfigFromFlags("", k8senv)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Get storage Policy")
+		vsanDefaultStoragePolicyName := "vSAN Default Storage Policy"
+		ginkgo.By(fmt.Sprintf("storagePolicyName: %s", vsanDefaultStoragePolicyName))
+		profileID := e2eVSphere.GetSpbmPolicyID(vsanDefaultStoragePolicyName)
+		log.Infof("Profile ID :%s", profileID)
+		scParameters := make(map[string]string)
+		scParameters["storagePolicyID"] = profileID
+		vsanDefaultStorageClass := "vsan-default-storage-policy"
+		err = client.StorageV1().StorageClasses().Delete(ctx, vsanDefaultStorageClass, metav1.DeleteOptions{})
+		if !apierrors.IsNotFound(err) {
```

Comment:
In this case I am deleting storage class. If storage class is already deleted or not present I will be getting not found error  and ignoring , it wont enter if block ( if  !apierrors.IsNotFound(err) ), For other errors it will enter if block and throws exception.